packages/gatsby-plugin-gatsby-cloud/src/components/IndicatorButton.js

```diff
@@ -81,35 +81,25 @@ const spinnerIcon = (
 )
 
 const IndicatorButton = ({
-  toolTipOffset,
-  isFirstButton,
-  tooltipText,
+  buttonIndex,
+  tooltipContent,
   overrideShowTooltip = false,
-  tooltipLink,
-  tooltipIcon,
-  tooltipLinkImage,
   iconSvg,
   onClick,
   showSpinner,
   active = false,
   testId,
-  onMouseOver,
+  onMouseEnter,
 }) => {
   const [showTooltip, setShowTooltip] = useState(false)
+  const isFirstButton = buttonIndex === 0
```

Comment:
Can't we use a different prop to proxy this, like tooltipContent or onMouseEnter? If not I would suggest going for `isHoverable `or something. Adding cheeky side-effects like these makes debugging more difficult.
```suggestion
```